test/v1beta1_samples/buildstrategy_samples.go

```diff
@@ -205,6 +215,14 @@ spec:
       volumeMounts:
         - name: varlibcontainers
           mountPath: /var/lib/containers
+  parameters:
+    - name: dockerfile
+      description: The path to the Dockerfile to be used for building the image.
+      type: string
+      default: "Dockerfile"
```

Comment:
Is not needed here, right?

```suggestion
```

Reply:
it is, although the parameter isn't used in the strategy itself.